src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -42,41 +40,38 @@ public class AddCommand extends Command {
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_NRIC + "S1231231D "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
-            + PREFIX_TRIAGE + "1 "
-            + PREFIX_TAG + "Pacemaker";
+            + PREFIX_TRIAGE + "1"
+            + PREFIX_TAG + "Pacemaker "
+            + PREFIX_TAG + "High Blood Pressure";
```

Comment:
This shouldn't work in our current implementation right? I thought our tags can't have whitespaces.

Reply:
Was a typo mb. resolved.